outputs.tf

```diff
@@ -13,21 +8,26 @@ output "cluster_resource_id" {
   description = "The RDS Cluster Resource ID"
 }
 
+output "cluster_identifier" {
```

Comment:
I know the alphabet is pretty difficult to get right, but `cluster_resource_id` really comes after `cluster_identifier` 😉